kai/service/incident_store/__init__.py

```diff
@@ -1,11 +1,9 @@
 from .incident_store import Application, IncidentStore, Solution
-from .psql import PSQLIncidentStore
-from .sqlite import SQLiteIncidentStore
 
 __all__ = [
     "IncidentStore",
     "Solution",
-    "PSQLIncidentStore",
+    "PSQLIncidentStoreBackend",
```

Comment:
What's the motivation for the name change here?

Reply:
I wanted it to be clear that the IncidentStore and postgres have a has-a relationship, instead of an is-a relationship. 

Though, I'm not sure if `__all__` is really needed in this `__init__.py` anymore.